docs/docs/reference/dropped-features/wildcard-init.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: doc-page
+title: "Dropped: wildcard initializer"
+---
+
+The syntax
+```scala
+  var x: A = _
+```
+that was used to indicate an uninitialized field, has been dropped.
+At its place there is a special value `uninitialized` in the `scala.compiletime` package. To get an uninitialized field, you now write
```

Comment:
Why need an import for this language feature at all?

Reply:
Because it's a very special case. Normally, fields should be initialized. In the special cases where that's no the case and import is in order. That's not for novices, so knowing about compiletime is OK.